doc/theoretical_description_regression.rst

```diff
@@ -178,6 +178,38 @@ methods and emphasizes their main differences.
    :width: 800
 
 
+7. The jackknife+-after-bootstrap method
+========================================
+In order to reduce the computational time, and get more robust predictions, 
+one can adopt a bootstrap approach instead of a leave-one-out approach, called 
+the jackknife+-after-bootstrap method, offered by Kim and al. [2].
+
+By analogy with the CV+ method, estimating the prediction intervals with 
+jackknife+-after-bootstrap is performed in four main steps:
+
+- We resample the training set with replacement (boostrap) :math:`K` times, 
+where :math:`K` is supposed to be drawn randomly from a fixed integer 
+:math:`K\_`, according to a binomial law (see [2]), and thus we get the 
```

Comment:
In the code, K is not drawn randomly but is a user-defined parameter, right ? If so, could you drop the "supposed..." part ?

Reply:
Right, even if the user can drawn it randomly if he wishes
I delete these few lines